scripts/create_registry_db.py

```diff
@@ -2,9 +2,9 @@
 import sys
 import enum
 import argparse
-from sqlalchemy import Column, Integer, String, DateTime, Boolean, Index
+from sqlalchemy import Column, Integer, String, DateTime, Boolean, Index, Float
 from sqlalchemy import ForeignKey, UniqueConstraint, Enum
-from dataregistry.db_basic import create_db_engine, TableCreator, ownertypeenum
+from dataregistry.db_basic import create_db_engine, TableCreator, ownertypeenum, datatypeenum
```

Comment:
`datatypeenum` suggests values like float, int, etc. rather than the values you have in mind. Can we find a more accurate name? Internally I see `dataset_type` which is  better, but still not very specific. `dataset_organization` would be better still but it is awfully long. Maybe `dataset_org` or even `data_org`? This comment applies to all the places where datatypeenum, data_type, etc. appear in your new code.

Reply:
I've updated the wording of the variable to dataset_organization in the code (data_org in the dataset table)